src/server/routes/entity/edition.js

```diff
@@ -339,7 +339,7 @@ function transformNewForm(data) {
 	);
 
 	let releaseEvents = [];
-	if (data.editionSection.releaseDate.year) {
+	if (data.editionSection.releaseDate && data.editionSection.releaseDate.year) {
```

Comment:
I think this would've worked also `if(data.edition.releaseData){..` because year is an empty string by default and will always be present.
But I think it's better to be safe :)

Reply:
Indeed. I think also if by any chance you get something other than a {year,month,day} object (like a date string), calling dateObjectToISOString with it will throw an error.